src/main/java/lsd/format/JsonConverter.kt

```diff
@@ -0,0 +1,16 @@
+package lsd.format
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.core.type.TypeReference
+import lsd.format.config.log
+import lsd.format.json.objectMapper
+
+fun convertJsonStringToMap(document: String?): Map<String, Any> =
+    document?.let {
+        try {
+            objectMapper.readValue(it, object : TypeReference<Map<String, Any>>() {})
+        } catch (e: JsonProcessingException) {
+            log().trace(e.message)
```

Comment:
wondering if worth considering bumping up to debug level?  (picturing that people would debug to figure out why the values are empty)